src/microbe_stage/Microbe.cs

```diff
@@ -1382,6 +1382,12 @@ internal void OnColonyMemberRemoved(Microbe microbe)
             RevertNodeParent();
             ai?.ResetAI();
 
+            // Colony reset at the end, to allow for previous operations on it.
+            Colony = null;
+
+            // Checks if the cell is ready to reproduce (allows editor access on unbind).
+            if (allOrganellesDivided)
+                OnReproductionStatus?.Invoke(this, true);
```

Comment:
A little nitpick but it's nice to have a whitespace separating the `return` and this.